docs/src/direct_sum.md

```diff
@@ -220,17 +184,17 @@ julia> p = ModuleHomomorphism(N, N, [3,4] .* basis(N));
 julia> q = ModuleHomomorphism(N, N, [5,7] .* basis(N));
 
 julia> phi = ModuleHomomorphism(D, D, [p 0; 0 q])
-Module homomorphism with
-Domain: DirectSumModule over Rationals
-Codomain: DirectSumModule over Rationals
+Module homomorphism
+from DirectSumModule over Rationals
+to DirectSumModule over Rationals
 
 julia> r = ModuleHomomorphism(N, D, [2,3] .* gens(D)[1:2])
-Module homomorphism with
-Domain: Vector space of dimension 2 over Rationals
-Codomain: DirectSumModule over Rationals
+Module homomorphism
+from Vector space of dimension 2 over Rationals
+to DirectSumModule over Rationals
 
 julia> psi = ModuleHomomorphism(D, D, [r, r])
-Module homomorphism with
-Domain: DirectSumModule over Rationals
-Codomain: DirectSumModule over Rationals
+Module homomorphism
+from DirectSumModule over Rationals
+to DirectSumModule over Rationals
```

Comment:
IMHO these should be indented like this:
```suggestion
Module homomorphism
  from DirectSumModule over Rationals
  to DirectSumModule over Rationals
```
just as we do for e.g.
```
Fraction field
  of Univariate Polynomial Ring in x over Integers
```

Reply:
I get the following single-line printed form:
```
julia> P,x = PolynomialRing(ZZ,"x");
julia> FractionField(P)
Fraction field of Univariate Polynomial Ring in x over Integer Ring
```

In the hand-typed examples, the indentation was by 2 spaces.  Is the amount of indentation determined by a global somewhere?

Reply:
Which "hand-typed examples" are you referring to? From the new printing "manifesto" in the OSCAR manual?

Is your example done with this PR? Because for me the output with this PR looks just as I copy&pasted it above -- in particular it is on two lines, with the second one indented. While what you pasted is on a single line.

Reply:
It is hardcoded.

Reply:
Oops!  Sorry.  It was with the standard version in Oscar... I'd forgot to notice that this was a PR.
As I wrote above it would be handy to have a standard/global way of inserting indentation so that it appears uniformly everywhere; perhaps a function or a global constant that returns the correct number of spaces?

Reply:
We will borrow ideas from https://github.com/KristofferC/IOIndents.jl and have an IO stream with indentation.